docs/01-overview/02-main-areas/service-mesh/smsh-01-default-istio-setup-in-kyma.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Default Istio setup in Kyma
+---
+
+Istio in Kyma is installed with the help of the `istioctl` tool. The tool is driven by a configuration file containing an instance of [IstioOperator](https://istio.io/docs/reference/config/istio.operator.v1alpha1/) custom resource.
+
+## Istio components
+
+This list shows the available Istio components and the components enabled in Kyma:
+
+| Component | Enabled |
```

Comment:
I'm a bit confused with this table... aren't Grafana, Prometheus, Tracing, and Kiali used in Kyma? 🤔

Reply:
These are kind of Istio addons - sample configurations of the components for Istio. We don't use them as we have our own. :)